lib/hocon/impl/config_number.rb

```diff
@@ -29,6 +29,23 @@ def transform_to_string
     @original_text
   end
 
+  def int_value_range_checked(path)
+    l = long_value
+
+    # guess we don't need to bother with this ...
```

Comment:
Also agreed. I think we should add a comment saying we don't need to check whether or not this is out of range due to the way Ruby handles Integers, and delete the commented-out code.